src/extension.ts

```diff
@@ -3,49 +3,77 @@ import SettingsHelper from "./settingsHelper";
 import ContributedCommands from "./commands";
 import Config from "./services/config";
 import ExtensionHelper from "./services/extensions";
+import * as liveShare from "./services/liveShare";
```

Comment:
Most of the churn in this file are refactorings that allowed the profile selection code to be re-used.